### PR TITLE
SWIK-1920 Internationalisation

### PR DESCRIPTION
**Pull request by rmeissn**
Please test and review these changes - social providers are not testable on localhost.

### CODE REVIEW

File: components/User/UserProfile/Integrations.js [hunk 1]
```diff
@@ -338,15 +413,15 @@ class Integrations extends React.Component {
                           <div className="two wide column">
                             {
                               (github) ? (
-                                <button className={github_disable_classes} name="github" onClick={this.handleDisable.bind(this)} >Disable</button>
+                                <button className={github_disable_classes} name="github" onClick={this.handleDisable.bind(this)} ><FormattedMessage id='Integration.disableGithub' defaultMessage='Disable'/></button>
                               ) : (
-                                <button className={github_enable_classes} name="github" onClick={this.handleEnable.bind(this)} >Enable</button>
+                                <button className={github_enable_classes} name="github" onClick={this.handleEnable.bind(this)} ><FormattedMessage id='Integration.enableGithub' defaultMessage='Enable'/></button>
```
Comment (TBoonX) [inline components/User/UserProfile/Integrations.js]: Somehow this breaks the event transmitted to the function. On master the button is working.

File: components/Login/ResetPassword.js [hunk 2]
```diff
@@ -56,12 +79,46 @@ class ResetPassword extends React.Component {
     }
 
     componentDidUpdate() {
+        const messages = defineMessages({
```
Comment (TBoonX) [inline components/Login/ResetPassword.js]: Why do you initialize this static content on each function call? Initializing it in the constructor and storing under this is what I would suggest.

Comment (TBoonX) [inline components/Login/ResetPassword.js]: Why do you initialize this static content on each function call? Initializing it in the constructor and storing under this is what I would suggest.

File: components/User/UserProfile/Integrations.js (hunk 1 above)
Comment (TBoonX) [inline components/User/UserProfile/Integrations.js]: Somehow this breaks the event transmitted to the function. On master the button is working.

File: components/Login/ResetPassword.js (hunk 2 above)
Comment (rmeissn) [reply components/Login/ResetPassword.js]: Never tested storing the variable in "this.state"...just oriented myself as of the provided examples from confluence (where this is also initialised in functions).

Comment (TBoonX) [reply components/Login/ResetPassword.js]: Hm OK. No need to change it.

File: components/User/UserProfile/Integrations.js (hunk 1 above)
Comment (rmeissn) [reply components/User/UserProfile/Integrations.js]: Fixed by recent commit

**Pull request merged**